src/hooks/useStore.js

```diff
@@ -0,0 +1,101 @@
+import create from 'zustand';
+import produce from 'immer';
+import { persist } from 'zustand/middleware';
+import { nanoid } from 'nanoid';
+
+const useStore = create(
+	persist(set => ({
+		cardlistzustand: [
+			{
+				id: nanoid(),
+				title: 'Angebot 1',
+				adress: 'Blumenfeld 05',
+				datum: '13-10-2022',
+				zeitraum: '17:15-17:30',
+				AnzahlFlaschen: '18',
+				Notiz: 'Flaschen stehen im Treppenhaus',
+				foto: {
+					url: 'https://images.unsplash.com/photo-1653059959899-70ab4e464401?ixlib=rb-1.2.1&raw_url=true&q=80&fm=jpg&crop=entropy&cs=tinysrgb&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=987',
+				},
```

Comment:
As already mentioned in a comment above I would recommend to name everything in english: 

```suggestion
				date: '13-10-2022',
				timeFrame: '17:15-17:30',
				amountBottles: '18',
				notes: 'Flaschen stehen im Treppenhaus',
				photo: {
					url: 'https://images.unsplash.com/photo-1653059959899-70ab4e464401?ixlib=rb-1.2.1&raw_url=true&q=80&fm=jpg&crop=entropy&cs=tinysrgb&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=987',
				},
```